pkg/testutil/well_known_generation_test.go

```diff
@@ -137,7 +137,7 @@ func createWellKnownDIDConfiguration(tt *testing.T, didResponse *did.CreateDIDRe
 	credToken := keyaccess.JWT(signedTokenBytes).Ptr()
 
 	wellKnown := map[string]interface{}{
-		"@context":    "https://identity.foundation/.well-known/did-configuration/v1",
+		"@context":    "https://tbd.website/.well-known/did-configuration/v1",
```

Comment:
this is 404 btw

Reply:
Good point. I'm going to leave as is. This file seems to be ignored in tests, and appears to be meant as an example only.